rules/windows/defense_evasion_execution_suspicious_explorer_winword.toml

```diff
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an instance of Explorer or Winword starting after being renamed or from a non-standard path. This is uncommon
+behavior and may indicate an attempt to evade defenses via side loading a malicious DLL within the memory
+space of one of those processes. 
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Potential DLL SideLoading via Trusted Microsoft Programs"
```

Comment:
Could probably make this a little more specific

Reply:
wanted to keep it open for future additions but its Ok for now renamed it to Potential DLL SideLoading via Explorer or WinWord

Reply:
FYI updated the rule with 2 more exes (were discussed in slack), w3wp.exe and Dism.exe, rule name is set back to generic to keep it open for more binaries in the future.